doc/sites.rst

```diff
@@ -96,7 +96,10 @@ site using the :doc:`generate_sites_module stanza
     (sites mygui))
 
 The generated module `mysites` depends on the library `dune-site` provided by
-Dune.
+Dune. If :doc:`implicit_transitive_deps
+</reference/dune-project/implicit_transitive_deps>` is set to false in your
+`dune-project` file, `dune-site` must be explicitly declared in the `depends`
+stanza of your package.
```

Comment:
Dune never uses `depends` itself so why must it be declared?

I think a better wording might be to state that `dune-site` should be added as a dependency if it is used (and that is true for opam files as well as the `depends` field).  `implicit_transitive_deps true` makes it work accidentally, but the dependency declaration is still wrong.